Sources/MapboxCoreNavigation/CoreConstants.swift

```diff
@@ -187,6 +187,13 @@ public extension Notification.Name {
      The user info dictionary contains the key `MapboxNavigationService.NotificationUserInfoKey.locationAuthorizationKey`.
     */
     static let locationAuthorizationDidChange: Notification.Name = .init(rawValue: "LocationAuthorizationDidChange")
+    
+    /**
+     Posted when `RouteController` detects the road name.
+     
+     The user info dictionary contains the key `RouteController.NotificationUserInfoKey.roadNameKey`.
+     */
+    static let routeControllerRoadName: Notification.Name = .init(rawValue: "RouteControllerRoadName")
```

Comment:
A notification name should describe something that has happened. In this case, the event is posted whenever the road name _changes_, so the name should be `currentRoadNameDidChange`. (This is also nice because the notification could be posted by PassiveLocationManager instead.)